README.md

```diff
@@ -127,132 +132,19 @@ There are several ways to [access the dashboard](https://discourse.pi-hole.net/t
-
-<p align="center">
-<a href=https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debugs><img src="https://assets.pi-hole.net/static/debug-token.png"></a>
-</p>
-
-You should be able to resolve most issues using the provided FAQ links, but we're always happy to help out if you'd like assistance!
-
-### Web Interface Settings
-The settings page lets you control and configure your Pi-hole. You can do things like:
-
-- view networking information
-- flush logs or disable the logging of queries
-- [enable Pi-hole's built-in DHCP server](https://discourse.pi-hole.net/t/how-do-i-use-pi-holes-built-in-dhcp-server-and-why-would-i-want-to/3026)
-- [manage block lists](https://discourse.pi-hole.net/t/how-do-i-add-additional-block-lists-to-pi-hole/259)
-- exclude domains from the graphs and enable privacy options
-- configure upstream DNS servers
-- restart Pi-hole's services
-- back up some of Pi-hole's important files
-- and more!
-
-<p align="center">
-<img src="https://assets.pi-hole.net/static/settings-page.png">
-</p>
-
-### Built-in DHCP Server
-Pi-hole ships with a [built-in DHCP server](https://discourse.pi-hole.net/t/how-do-i-use-pi-holes-built-in-dhcp-server-and-why-would-i-want-to/3026). This allows you to let your network devices use Pi-hole as their DNS server if your router does not let you adjust the DHCP options.
-
-One nice feature of using Pi-hole's DHCP server if you can set hostnames and DHCP reservations so you'll [see hostnames in the query log instead of IP addresses](https://discourse.pi-hole.net/t/how-do-i-show-hostnames-instead-of-ip-addresses-in-the-dashboard/3530). You can still do this without using Pi-hole's DHCP server; it just takes a little more work. If you do plan to use Pi-hole's DHCP server, be sure to disable DHCP on your router first.
-
-<p align="center">
-<a href=https://discourse.pi-hole.net/t/how-do-i-use-pi-holes-built-in-dhcp-server-and-why-would-i-want-to/3026><img src="https://assets.pi-hole.net/static/piholedhcpserver.png"></a>
-</p>
-
-### Real-time Statistics
-Using [chronometer2](https://github.com/pi-hole/pi-hole/blob/master/advanced/Scripts/chronometer.sh), you can view [real-time stats](https://discourse.pi-hole.net/t/how-do-i-view-my-pi-holes-stats-over-ssh-or-on-an-lcd-using-chronometer/240) via `ssh` or on an LCD screen such as the [2.8" LCD screen from Adafruit](http://amzn.to/1P0q1Fj).
-
-Simply run `pihole -c` for some detailed information.
-
-<p align="center">
-<img src="https://assets.pi-hole.net/static/chrono1.jpg">
-<sub><a href="https://www.reddit.com/r/pihole/comments/6ldjna/pihole_setup_went_so_well_at_home_for_the_1st/">Image courtesy of /u/super_nicktendo22</a></sub>
-</p>
```

Comment:
Was all of this moved somewhere?

Reply:
Core features were moved to https://github.com/pi-hole/pi-hole/wiki/Core-Function-Breakdown, specifically, [this link for Chronometer](https://github.com/pi-hole/pi-hole/wiki/Core-Function-Breakdown#chronometer).

Reply:
I mean the web feature information.

Reply:
Nothing specific for the Web Interface yet. I'm thinking I'll put in a bullet list that includes *what else* the Web Interface provides without duplicating the information already present in the CLI feature list. In the future, that list can then link to the equivalent Web Interface breakdown article that includes details and pictures about every feature.